pkg/projects/Microsoft.Net.UWPCoreRuntimeSdk/files/Microsoft.Net.UWPCoreRuntimeSdk.props

```diff
@@ -0,0 +1,5 @@
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <UseAppLocalCoreCLR>true</UseAppLocalCoreCLR>
+  </PropertyGroup>
```

Comment:
This comes down to "VS magic". VS will look through all of the NuGet packages that are being referenced and load all of the *.props and *.targets files that are in the path <PackageRoot>\build. So trust that this gets imported correctly.